.github/workflows/publishbot.yml

```diff
@@ -10,16 +10,9 @@ jobs:
     runs-on: ubuntu-latest
     steps:
       - run: echo "Workflow allowed to start"
-  build:
```

Comment:
Since the next jobs aren't run in parallel, there's no point building in advance.